### ISSUE TITLE
Grafana not runnable on MacOS?

### ISSUE THREAD

**Issue by AAverin** [hashicorp/nomad-pack-community-registry#154]
https://github.com/hashicorp/nomad-pack-community-registry/blob/ef86104fbcbc61eac8312a7fe5cf0e330e5f30f0/packs/grafana/templates/grafana.nomad.tpl#L8

Constraint set to linux makes it impossible to run Grafana on MacOS and test monitoring setup in a local cluster.

**Comment by jrasell** [comment]
Hi @AAverin, this seems to be because the network mode is currently hard coded to bridge which is only supported on Linux machines. In order to remove the constraint, this would need to be made configurable.

**Comment by AAverin** [comment]
@jrasell Yes, this part I have figured out and made a custom pack to make it work on my local machine.
Just wonder if something should be done in this default registry to keep it more flexible for everyone using `nomad-pack`
